docs/content/interop.md

```diff
@@ -25,83 +25,27 @@ You need to use the ICG to generate `.schema` files and type bindings for any ne
 ### How to use the ICG
 As the Interop Code Generator is an plugin for the Unreal toolbar, to use it you must first:
 *  build your game project to access the SpatialOS fork of Unreal Engine as well as the Unreal GDK. (You do this as part of the [installation and setup](../setup-and-installing.md), swapping the “Starter Project” and its file location for your game’s name and file location.)
-* set up a `DefaultEditorSpatialGDK.ini` file. See  [Setting up the Interop Code Generator](#Setting-up-the-Interop-Code-Generator), below.
+* List the classes that you want create typebindings for in the `SpatialGDKEditorSettings` from within the editor. See  [Setting up the Interop Code Generator](#Setting-up-the-Interop-Code-Generator), below.
 
 The SpatialOS build of Unreal has an **Interop Codegen** button; on the SpatialOS Unreal GDK toolbar, select **Interop Codegen** to trigger the ICG process:
 
 ![Interop Codegen button on toolbar](../assets/screen_grabs/codegen.png)
 
 ## Setting up the Interop Code Generator
 
-In order for the ICG to run correctly, you need to add any new user-defined class, along with its dependencies, to a `DefaultEditorSpatialGDK.ini` file before you run it.
+In order for the ICG to run correctly, you need to add any new user-defined class, along with its dependencies, in the `SpatialOS Unreal GDK` settings in the Unreal Engine 4 Editor tab before you run it.
 
 Using the Unreal GDK Starter Project as an example, to do this:
-1. Locate the `DefaultEditorSpatialGDK.ini` file in the Starter Project repository which you cloned during installation and setup - it’s at `Game/Config/DefaultEditorSpatialGDK.ini`.
-1. Open the file in your editor and add your user-defined class as shown in the file snippet and file description below.
+1. Go to ** Edit > Project Settings... > SpatialOS Unreal GDK > Toolbar **.
+1. In the entry `Classes to generate typebindings for`, add your user-defined class and the include dependencies that the typebinding require to compile.
```

Comment:
the typebindings require 
OR
the typebinding requires

(Think it's the first one?!)

Reply:
If "Classes to generate typebindings for" is on-screen text, part of the GUI, then it's bold.

We need to update the screen grab of toolbar.md#Settings once this is all in.
Will add to my TODO lists.